### PR TITLE
Update usage example

### PR DESCRIPTION
**Pull request by frostney**
- Using `React` directly from `react-native` has been deprecated and it is not available since React Native 0.26
- Submodules from `react-native` should rather be imported by accessing the property from the imported `react-native` object. Requiring via the internal module shortcut is considered bad practice outside of `react-native` codebase itself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,8 +73,9 @@ This method returns a boolean value as a callback indicating whether or not the
 ## Usage Example
 
 ```js
-var React = require('react-native');
-var Composer = require('NativeModules').RNMessageComposer;
+import React from 'react';
+import { NativeModules } from 'react-native';
+const Composer = require('NativeModules').RNMessageComposer;
```
Comment (dozoisch) [inline README.md]: you import NativeModules, but then require it the next line!

Comment (frostney) [inline README.md]: Sorry about that. I just push changes updating the this line.

**Comment by anarchicknight** [comment]
will hopefully get a chance to look at this on the weekend

**Comment by anarchicknight** [comment]
Going to close this as your latest PR requires further changes to this file and can be handled in that PR

**Pull request closed without merging**